config/yugabyte_tpcc_config_1.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>

```

Comment:
I would recommend making the config file name more descriptive. For example, `config_workload_1`, `config_workload_2` etc. We can also remove the `yugabyte_tpcc` prefix since this YB tpcc repo anyway.

Lets also add one more config to run all workloads (similar to what oltpbench has) - `config_workload_all`

Reply:
Done.